script/Ciel/Day 10/10_01_CIEL10_4_2C.txt

```diff
@@ -129,120 +139,131 @@ I see.
 -- Page 229, Offset 27693.
 -- 教会の偉い人にもお礼を言っておいてください。
-Please, do convey my gratitude to the higher-ups of the Church as well.
+Please convey my gratitude to the higher-ups at the Church, too.
 }
 [sha:89e2eb9e9793800e3112924095900e37d03915e3]{
 -- Page 230, Offset 27694.
 -- 「ヴローヴとの戦いで用いた武装は殲滅用のものです。
-"What I used to fight against Vlov is a weapon used for eradication.
+"The weapon I used in the fight against Vlov is tailored for eradication.
 }
 [sha:81bd76753276839ea4519cd33e2f1b0a4a6a606e]{
 -- Page 230, Offset 27695. Mods: @n.
 -- 戦場が人々の目の届かぬ場所であり、
-Only when the battlefield is out of people's sight―――
+// Sentence ordering here kinda shit in EN if 1:1, so this triplet has been
+// munged and manually linebroken so that it renders like one normal sentence
+I only use it when there's no chance of bystanders
 }
 [sha:5f84e5703ae2f0a157757244cb96a2ee60297843]{
 -- Page 230, Offset 27696. Mods: @n.
 -- 対象が救済の余地のない害獣である場合のみ、
-Only against a wild beast with no hope of salvation―――
+witnessing the battle, and my target is a wild beast
 }
 [sha:9041f14d7b224a96be45072da49ef296cb61d2fc]{
 -- Page 230, Offset 27697.
 -- 選択されます」
-Is it chosen."
+that is already beyond salvation."
 }
 [sha:bc7474b2d65d844695dcd06bc8ad44eaa2987017]{
 -- Page 231, Offset 27698.
 -- 「……とはいえ、遠野くんに助け起こされた時には半壊していましたけどね。
-"That being said... It was half-destroyed when you saved me, Tohno-kun.
+"That being said... it had already been partially destroyed when you helped me up back then.
 }
 [sha:ed9dfb148f7707292d3d8606144d25be949805ae]{
 -- Page 231, Offset 27699.
 -- わたしの魔力が尽きないかぎり破損しない甲冑だったのですが、あの極寒と重槍の前に音を上げてしまったようです」
-The armor was supposed to never be damaged as long as I still had magical energy, but it looks like it relented in the face of that extreme cold and his heavy spear."
+That armor was supposed to be impregnable so long as I still had magical energy,
+ but unfortunately it gave up the ghost in the face of that extreme cold and his massive lance."
 }
 [sha:a7af7ada711f0903f11951bff9a47abd4a0fddbe]{
 -- Page 232, Offset 27700.
 -- 「遠野くんが見たあのスタイルは、甲冑が剥がれた後のインナーなんです。
-"The style that you saw, Tohno-kun, is in fact the inner part hidden behind the armor.
+"The outfit you saw me in back then was actually the undergarments;
+ the outer armor had already been stripped away.
 }
 [sha:3a2b5e00a7ae920c3c070204f5f641512f7fc207]{
 -- Page 232, Offset 27701.
 -- ヴァージンペインはセブンのところで修理中なので、しばらくは使えません」
-// Is it repaired 'by Seven', 'where Seven is', 'with Seven' or something else?
-Virgin Pain is currently being repaired by Seven, so I won't be able to use for now."
+// G: Is it repaired 'by Seven', 'where Seven is', 'with Seven' or something else?
+// R: I think Seven's humansona is mostly a goof, and she mentions Seven needing maintenance in the prev file,
+// so assuming this is undergoing maintenance in the same place/manner as Seven?
+Virgin Pain is currently being repaired along with Seven, so I won't be able to use it again for a while."
```

Comment:
Seven is the main component of 第七聖典. You sort of see it later in the route. She's got it up on a building. It can perform automatic reconfiguration of the other armaments that are a part of it. Nasu described it as a factory in an interview. That's all to say that you kinda should just be a bit literal here.

>Virgin Pain is currently being repaired at where Seven is, so I won't be able to use it again for a while.

Reply:
Sure, first one works for me.